tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -745,30 +760,26 @@ void TangoRosNode::PublishStaticTransforms() {
 }
 
 void TangoRosNode::OnPoseAvailable(const TangoPoseData* pose) {
-  if (publish_pose_on_tf_ ||
-      start_of_service_T_device_publisher_.getNumSubscribers() > 0 ||
-      area_description_T_start_of_service_publisher_.getNumSubscribers() > 0) {
```

Comment:
To be able to express the origin of the map in the area description frame we need to know the transform `area_description_T_start_of_service`, which is filled in only after these checks. I removed them so that we are sure that `area_description_T_start_of_service` is always known.

These checks were here to avoid slowing down the callback if no one is using the pose data. But I don't think it is really necessary for the pose callback.
Consequently I also removed the corresponding checks in the `StopPublishing` method, which were there to unlock the publishing thread in case we can not reach the `notify_all()` call inside the Tango callbacks.